### ISSUE TITLE
Built-in Costume Sizing Issues

### ISSUE THREAD

**Issue by maryfries** [jmoenig/Snap--Build-Your-Own-Blocks#1682]
Some of the costumes are appearing at a larger size than ever before...

![costume-sizing](https://cloud.githubusercontent.com/assets/6549787/23876751/4d3808a8-0815-11e7-9424-287f729e6528.png)

**Comment by jguille2** [comment]
Hi Mary,

Maybe you applied a resizing action before?
![resizingblocks](https://cloud.githubusercontent.com/assets/2926315/23880400/cf651a62-0853-11e7-98b6-e15c80d41b69.png)
You can check _size reporter_ to see current value.
![sizereporter](https://cloud.githubusercontent.com/assets/2926315/23880451/0e34aa78-0854-11e7-9f1d-499ebe77f1fe.png)

Joan

**Comment by maryfries** [comment]
That is not the problem. Please try the "kai a" costume. It's now, suddenly, too big to match the other costumes.

**Comment by jmoenig** [comment]
Yes, this is definitely a real issue. These costumes clearly have been altered somehow at some time. @brianharvey , any idea?

**Comment by jguille2** [comment]
Hi @maryfries ,
I see now. I tested your issue... but I loaded 'boy1 standing' costume (it's the same picture!!!) and this is ok.

Now I see 'kai a' costume with the problem reported.

Then, we have a problem with duplicates (not exactly duplicates, because they have different formats). And yes, there is a relation with #1681 and Jens's request to Brian (and the format info) and also with #1666 (a proposal for a new costume picker view... this could help in this theme).

Now we have some 'duplicates': "kai a"(png) and "boy1 standing"(gif), "dog2" (png and svg), "bat1" (png and svg), "girl3"(gif) and "hannah"(png)... 

We need a bit of order.

Joan

**Comment by brianharvey** [comment]
@jmoenig  I have no idea.  All the costumes were uploaded at once in Dec; then a few of them (but not "kai a") in a batch Jan 15.  The new costumes are all over the place on height; I wonder if Scratch is normalizing them when loading?
